tgui/packages/tgui/interfaces/ArtGallery.js

```diff
@@ -0,0 +1,151 @@
+import { resolveAsset } from '../assets';
+import { useBackend, useLocalState } from '../backend';
+import { Button, NoticeBox, Section, Stack, Tabs } from '../components';
+import { Window } from '../layouts';
+
+export const ArtGallery = (props, context) => {
```

Comment:
pretty sure this UI is mostly copypasted from the ModPC art gallery. Use components properly and integrate the art gallery app under one component